src/utils/utils.jl

```diff
@@ -1,3 +1,7 @@
+using Base.Filesystem
```

Comment:
You shouldn’t have to import this.

We follow a convention where all imports occur in the main package file.